include/picongpu/param/species.param

```diff
@@ -19,9 +19,25 @@
 
 /** @file
  *
- * Forward declarations for speciesDefinition.param in case one wants to use
- * the same particle shape, interpolation, current solver and particle pusher
- * for all particle species.
+ * Particle shape, field to particle interpolation, current solver, and particle pusher
+ * can be declared here for usage in @see speciesDefinition.param.
+ * 
+ * @remark
+ * Macroparticle shape naming is according to
+ *    Hockney R, Eastwood JW. Computer simulation using particles. IOP Pubslishing Ltd (1988). p. 144.
+ * 
+ * The name of macroparticle shape is defined by the visual form of its cloud shape S,
+ * which describes a macroparticle's charge density distribution.
+ * However, the charge assigned to a mesh point due to a distribution of overlapping finite-sized
+ * macroparticles is obtained by evaluating and summing the assignment function W for all particles.
+ * The assignment function is obtained by convolving the cloud shape with the top-hat function.
+ * PIConGPU only implements the assignment functions.
+ *
+ * @warning
+ * The higher order shape names are redefined with release 0.6.0 in order to provide a consistent naming:
+ *     -> PQS is the name of the 3rd order shape (instead of PCS)
+ *     -> PCS is the name of the 4th order shape (instead of P4S)
+ *     -> P4S does not exist anymore
```

Comment:
I think this piece is not self-consistent. From the first sentence a reader may assume everything is about cloud shapes. But these `3rd` and `4th order shape` are **not** shape orders. Rather, they are assignment function orders

Reply:
I think the text should be corrected to have `3rd order assignment function` instead of `3rd order shape`